src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -36,7 +36,7 @@
   <Target
     Name="_RunForCore"
     DependsOnTargets="_CreateResponseFileForMvcRazorPrecompile"
-    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp'">
+    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' AND '$(RuntimeIdentifier)'==''">
```

Comment:
I still see this property lingering around the sdk but I guess the behaviour changed? https://github.com/dotnet/sdk/blob/master/TestAssets/TestProjects/AppWithLibraryAndRid/App/App.csproj#L6? https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/buildCrossTargeting/Microsoft.NET.Sdk.targets#L62? Going to do a quick check to see what is using this.